internal/db/sqltest/tests/index/update_time.sql

```diff
@@ -0,0 +1,14 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+  select plan(1);
+  
+  select results_eq(
+    'select count(*)::bigint from get_deletion_tables()',
+    'select count(*)::bigint from pg_indexes where schemaname = ''public'' and indexname like tablename || ''_update_time_idx''',
```

Comment:
I don't want to be pedantic but this isn't _really_ testing that each deletion table "parent" has an update_time index. Can't we make a loop like we do for the deletion tables here: https://github.com/hashicorp/boundary/blob/c6b26fceaa7f62ca028bc4b05487f99f4ec6a2fd/internal/db/sqltest/tests/purge/deleted_table_tests.sql#L63-L83?

Reply:
> I don't want to be pedantic but this isn't _really_ testing that each deletion table "parent" has an update_time index. Can't we make a loop like we do for the deletion tables here:

I agree. This is not testing if the indexes exist. Any reason not to use pgTap's [`is_indexed`](https://pgtap.org/documentation.html#is_indexed) function?